README.md

```diff
@@ -159,7 +153,7 @@ Guest Operating System:
 
 ### Public
 * [containers@vmware.com](cna-storage <containers@vmware.com>)
```

Comment:
https://store.docker.com/plugins/e15dc9d5-e20e-4fb8-8876-9615e6e6e852?tab=description is also pointing to the same email id.

BTW, I haven't touched that line; not sure if it is showing into the diff